src/utils/selectors.ts

```diff
@@ -74,11 +74,18 @@ export const worldUsersByIdSelector: SparkleSelector<
  * Selector to retrieve venues from the Redux Firestore.
  *
  * @param state the Redux store
+ *
+ * @deprecated This selector requires all of the venues data in firebase to be loaded into memory. Find a different way.
+ * @debt Refactor all places that rely on this, then remove it from the codebase
  */
 export const venuesSelector: SparkleSelector<Record<string, AnyVenue>> = (
   state
 ) => state.firestore.data.venues || {};
 
+/**
+ * @deprecated This selector requires all of the venues data in firebase to be loaded into memory. Find a different way.
+ * @debt Refactor all places that rely on this, then remove it from the codebase
+ */
```

Comment:
[note, ?OOS?] This appears to be used in the following locations across the codebase (which may not necessarily rely on this specific connection):

```
src/types/Firestore.ts
  FirestoreData
    venues
      src/utils/selectors.ts
        venuesSelector
          makeVenueSelector
            src/pages/Admin/Admin.tsx

  FirestoreOrdered
    venues
      src/utils/selectors.ts
        orderedVenuesSelector
          src/components/organisms/UserProfileModal/UserProfileModal.tsx
          src/components/templates/Playa/Playa.tsx
          src/hooks/useRoom.ts
          src/pages/Admin/Admin.tsx
          src/pages/Admin/Admin_v2.tsx
          src/pages/Admin/VenueList/VenueList.tsx
```

Reply:
OOS, will handle this in a followup PR.